README.md

```diff
@@ -288,3 +288,19 @@ $ npm run build
 
 Following all the steps should generate the `own_home-5.6.4.zip`  Kibana plugin
 zip file under Own Home `build/` directory.
+
+
+## Removing the plugin
+
+### Remove plugin and delete kibana.yml
+
+```sh
+$ bin/kibana-plugin remove own_home
+$ rm config/kibana.yml
```

Comment:
I think this `rm config/kibana.yml` is dangerous. Because this file may contain the other important configs such kibana itself, elasticsearch, and other plugins.